backend/apps/owasp/management/commands/owasp_run_notification_worker.py

```
@@ -0,0 +1,295 @@
+                for message_id, data in result[1]:
+                    self.stdout.write(f"Recovering stuck message: {message_id}")
+                    try:
+                        self.process_message(data)
+                        redis_conn.xack(stream_key, group_name, message_id)
+                        self.stdout.write(f"Successfully recovered message {message_id}")
+                    except Exception:
+                        logger.exception("Failed to recover message %s", message_id)
+            else:
+                self.stdout.write("No stuck messages found.")
+        except Exception:
+            logger.exception("Error checking PEL for stuck messages")
```

Ruff (0.14.14)

[warning] No newline at end of file

Add trailing newline

(W292)